routers/google.controller.js

```diff
@@ -0,0 +1,31 @@
+const { userModel } = require('../database/models');
+
+class GoogleController {
+  constructor() {}
+  async findOrCreate(profile, callback) {
+    try {
+      const { email } = profile._json;
+      const displayName = email.substring(0, email.indexOf('@'));
+      const user = userModel.update(
+        email,
```

Comment:
I'm pretty sure that .update does not return user. Also I've told about `findOneAndUpdate`